##EEP_MAP
uint8  ModelNo1
uint8  ModelNo2
uint8  Version1
uint8  Version2
uint8  BaudRate
uint8  Reaserved1
uint8  ID
uint8  AckPolicy
uint8  AlarmLEDPolicy
uint8  TorquePolicy
uint8  Reserved2
uint8  MaxTemperature
uint8  MinVoltage
uint8  MaxVoltage
uint8  AccelerationRatio
uint8  MaxAccelerationTime
uint8  DeadZone
uint8  SaturatorOffset
uint16 SaturatorSlope
int8   PWMOffset
uint8  MinPWM
uint16 MaxPWM
uint16 OverloadPWMThreshold
uint16 MinPosition
uint16 MaxPosition
uint16 PositionKp
uint16 PositionKd
uint16 PositionKi
uint16 PositionFeedforward1stGain
uint16 PositionFeedforward2ndGain
uint16 VelocityKd
uint16 VelocityKi
uint8  LEDBlinkPeriod
uint8  ADCFaultCheckPeriod
uint8  PacketGarbageCheckPeriod
uint8  StopDetectionPeriod
uint8  OverloadDetectionPeriod
uint8  StopThreshold
uint8  InpositionMargin
uint8  Reserved5
int8   CalibrationDifference_L
int8   CalibrationDifference_H
